alarm/utils.py

```diff
@@ -33,3 +30,58 @@ def any_of_trade_pair_thresholds_is_broken(trade_pair):
             return True
 
     return False
+
+
+def create_thresholds_brakes_from_recent_candles_update(trade_pair):
+    thresholds = Threshold.objects.filter(trade_pair=trade_pair)
+    for threshold in thresholds:
+        ThresholdBrake.objects.create(threshold=threshold)
+
+
+def get_thresholds_brake_prices(trade_pair):
+    thresholds_brake_prices = ThresholdBrake.objects.filter(threshold__trade_pair=trade_pair).order_by(
+        '-happened_at').values_list(
+        'threshold__price', flat=True)
+    return thresholds_brake_prices
+
+
+def get_trade_pair_current_price(trade_pair):
+    last_candle = Candle.last_for_trade_pair(trade_pair)
+    return last_candle.high_price if last_candle else None
+
+
+def create_message_for_threshold_break(trade_pair):
```

Comment:
Create is usually referred to an object creation id db, I'd use get_message_for_threshold_break

Reply:
++